src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -559,3 +559,89 @@ def _patch(self, crd, crd_name, cr_name, func):
                 "Failed to update {}/{}: {}".format(crd_name, cr_name, e))
 
         return "Success"
+
+    def get_ceph_image(self):
+        image = "ceph/ceph:v14.2.6"
+
+        try:
+            api_response = self.coreV1_api.list_namespaced_pod(self.rook_env.namespace, label_selector="app=rook-ceph-mon", timeout_seconds=10)
+            if api_response.items:
+                image = api_response.items[-1].spec.containers[0].image
+        except ApiException as e:
+            log.warning("Error getting ceph image: {}, using <{}> as default".format(e, image))
+
+        return image
```

Comment:
another idea would be to fetch this from the CephCluster CR?

Reply:
Why is better to get it from the cluster CR?

Reply:
In case of an upgrade scenario, the pod image might be the old image. The CephCluster image should be the "correct" one.

Reply:
if it is the old image, there are lot of possibilities that the OSDs will have also the same old image, and what i want to do is to use a Ceph image already present in the node.
If the mon has an updated image, and the OSDs an older image, this should be updated, turn off/on a light will be even positive because we are going to download the new image before the upgrade procedure, doing this more fast.

I think that to use the mon image is the best  choice.

Reply:
ok with me. But then, can we please fail, instead of using `"ceph/ceph:v14.2.6"` by default?

Reply:
Ok. Done.